jsapp/js/ui.es6

```diff
@@ -98,6 +98,21 @@ class Modal extends React.Component {
       );
     }
   }
+  renderClose() {
+    if (this.props.title === 'Success!') {
```

Comment:
I feel like this `'Success!'` string should be put in a constant somehwere, as it is being used in two places, but plays an important role in code logic. Plus here you are checking `'Success!'` (english hardcoded text that will always be that), but in other places you are setting `t('Success!')` which potentially would be not in english :)

Reply:
Instead of hard coding it twice I think sending `isDuplicated` to `ui.es6` is a bit cleaner. Let me know what you think :) https://github.com/kobotoolbox/kpi/pull/2928/commits/39b51998b37ca3445012447084ce178e7b781ec2

Reply:
Love it :heart: